components/menu/test/menu.test.js

```diff
@@ -256,22 +254,19 @@ describe('d2l-menu', () => {
 					await super.getUpdateComplete();
 					return this._readyPromise;
 				}
-				makeReady() {
-					this._ready = true;
-					this._resolve();
-				}
 			}
 		);
 
-		const elem = await fixture(`
+		const elemPromise = fixture(`
 			<d2l-menu>
 				<d2l-menu-item text="fast"></d2l-menu-item>
 				<${delayedUpdateMenuItem}></${delayedUpdateMenuItem}>
 				<span>not a menu item</span>
 			</d2l-menu>
 		`);
```

Comment:
Had to rework how this "slow element" works since `fixture` now won't resolve until everything is ready anyway.